src/DataBus/Util/IAsyncTimer.cs

```diff
@@ -0,0 +1,11 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+
+    interface IAsyncTimer
+    {
+        void Start(Func<Task> callback, TimeSpan interval, Action<Exception> errorCallback);
+        Task Stop();
```

Comment:
Why does it need to return a `Task`? There's nothing inside the method that would require it, right?

Reply:
This is a copy paste from the core `IAsyncTask` / `AsyncTask`. We can change it, but I prefer just to leave it as is since the `AsyncTask` implementation allows the last loop to complete if it hasn't using this.

Eventually I would prefer the core team add a `Stop` method to `IDataBus` in the core and give us proper shutdown semantics, which would be a `Task` as everything else is.

Reply:
`IDataBus.Stop` would be nice.